content/2017/1/31/css-day.rst

```diff
@@ -0,0 +1,165 @@
+      Slides »
+    </a>
+
+.. _Miriam Suzanne: https://twitter.com/mirisuzanne/
+
+
+The In-Betweeners of Responsive Web Design by `Pralie Dutzel`_
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. raw:: html
+
+    <iframe width="853" height="480" src="https://www.youtube.com/DLMcBwwyZu0" frameborder="0" allowfullscreen></iframe>
+
+    <a href="http://praliedutzel.com/talks/betweeners-responsive-web-design/" class="btn">
+      Slides »
+    </a>
+
+.. _Pralie Dutzel: https://twitter.com/praliedutzel
+
+
+Style Guide-Driven Development: Documentation that Does Work by `Jan Jorgensen`_
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. raw:: html
+
+    <iframe width="853" height="480" src="https://www.youtube.com/jT66ITyme2Q" frameborder="0" allowfullscreen></iframe>
+
+.. _Jan Jorgensen: https://twitter.com/ramblinjan
+
+
+The Great State of Design by `Stacy Kvernmo`_
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. raw:: html
+
+    <iframe width="853" height="480" src="https://www.youtube.com/embed/AYXY11Bp2UI" frameborder="0" allowfullscreen></iframe>
+
+    <a href="http://www.slideshare.net/Funstacy/the-great-state-of-design-with-css-grid-layout-and-friends" class="btn">
+      Slides »
+    </a>
+.. _Stacy Kvernmo: https://twitter.com/stacykvernmo
+
+
+jQuery out, CSS in by `John Boere`_
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+John’s video is yet to be posted, but his slides are available. His talk was a
+case study looking into the benefits of switching from jQuery to CSS for their
+web and mobile applications.
+
+.. raw:: html
+
+    <a href="https://www.dropbox.com/s/u2sz0yfjl4753y7/1612_CSSday.pdf" class="btn">
+      Slides »
+    </a>
+
+.. _John Boere: https://twitter.com/cliffhangersolu
+
+
+Solving CSS at scale with your own UI framework by `CJ Cenizal`_
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. raw:: html
+
+    <iframe width="853" height="480" src="https://www.youtube.com/_ohLo7_5iRI" frameborder="0" allowfullscreen></iframe>
+
+    <a href="http://cenizal.com/projects/solving_css_at_scale/" class="btn">
+      Slides »
+    </a>
+
+.. _CJ Cenizal: https://twitter.com/TheCJCenizal
+
+
+The organizers and volunteers did a great job bringing the community together.
+
+    “Helping to plan CSSDay with my friends as a conference by developers, for
+    developers, has been one of the most rewarding and exciting things that
+    I've been able to do in my career. I’ve meet so many amazing and talented
+    people along the way. It has been an experience.” - Brad Westfall
+
+This was the second year `Brad Westfall`_ organized this event. We are very
+thankful for conference organizers, like Brad and his team, who post `videos`_
+of the sessions making this knowledge accessible for those who could not join
+us live. `Photos`_ from CSSDay are also posted.
+
+As always, we would love to discuss the content covered in these videos shared
```

Comment:
I change the message up here and there so I don't know if we want to make it completely built in?